packages/sdk-client-v3/src/utils/userAgent.ts

```diff
@@ -16,10 +16,10 @@ const isBrowser = (): boolean =>
 function getSystemInfo(): string {
   if (isBrowser()) return window.navigator.userAgent
 
-  const nodeVersion: string = process?.version.slice(1) || 'unknow' // unknow environment like  React Native etc
-  const platformInfo = `(${process.platform}; ${process.arch})`
+  const nodeVersion: string = process?.version?.slice(1) || 'unknow' // unknow environment like  React Native etc
```

Comment:
@ajimae Hey, thanks a lot for quick fix

Would you be able to fix typo as well please, on next planned release?

`unknow` => `unknown`